hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/metrics/S3GatewayMetrics.java

```diff
@@ -317,4 +317,52 @@ public long getHeadBucketSuccess() {
   public long getHeadKeySuccess() {
     return headKeySuccess.value();
   }
+
+  public long getGetBucketSuccess() {
+    return getBucketSuccess.value();
+  }
+
+  public long getGetBucketFailure() {
+    return getBucketFailure.value();
+  }
+
+  public long getCreateBucketSuccess() {
+    return createBucketSuccess.value();
+  }
+
+  public long getCreateBucketFailure() {
+    return createBucketFailure.value();
+  }
+
+  public long getDeleteBucketSuccess() {
+    return deleteBucketSuccess.value();
+  }
+
+  public long getDeleteBucketFailure() {
+    return deleteBucketFailure.value();
+  }
+
+  public long getGetAclSuccess() {
+    return getAclSuccess.value();
+  }
+
+  public long getGetAclFailure() {
+    return getAclFailure.value();
+  }
+
+  public long getPutAclSuccess() {
+    return putAclSuccess.value();
+  }
+
+  public long getPutAclFailure() {
+    return putAclFailure.value();
+  }
+
+  public long getListMultipartUploadsSuccess() {
+    return listMultipartUploadsSuccess.value();
+  }
+
+  public long getListMultipartUploadsFailure() {
+    return listMultipartUploadsSuccess.value();
```

Comment:
Sorry, I just noticed this typo:

```suggestion
  public long getListMultipartUploadsFailure() {
    return listMultipartUploadsFailure.value();
```